.travis.yml

```diff
@@ -42,7 +42,7 @@ jobs:
   include:
     - stage: flake8
       name: Flake8
-      script: docker-compose --log-level ERROR -f scripts/ci/docker-compose.yml run airflow-testing /app/scripts/ci/run-ci.sh
+      script: docker-compose --log-level ERROR -f scripts/ci/docker-compose-flake8.yml run airflow-testing /app/scripts/ci/run-ci.sh
```

Comment:
Instead of copying and pasting the compose file, could we use the existing compose file with the `--no-deps` flag?

Reply:
I've introduced your solution. I did not know about this trick.
Thanks for comment .